docs/getting_started.md

```diff
@@ -20,6 +26,12 @@ Docker image.
 
 Ubuntu on Windows WSL 2.0 seems to work for most things, but you may encounter errors with more complex operations. For instance, compiling Linux is known not to work in this environment. This is considered an experimental build.
 
+If you are using a newer version of Ubuntu (e.g., 20.04 LTS), you will want to update `setuptools`
+in Python if you plan to install the sdk (which is installed by `make prep` below).
+
+    sudo apt-get install python3 python3-pip
```

Comment:
Doesn't python3 ship with Ubuntu for 20.04?

Reply:
Yes, removing.